src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -83,7 +83,7 @@ type Gene = {
   name: string;
   stable_id: string;
   transcripts: Transcript[];
-  cross_references: CrossReference[];
+  external_references: CrossReference[];
```

Comment:
I have created `ExternalReference` in #368. Can we merge the two or is `CrossReference` a different type?

Reply:
I missed the implication of that in #368. Definitely the same type; but the CrossReference has been used more; and then there is an ExternalReference component. I'm not quite sure what to do. I am tempted to unify everything under the CrossReference name; but at the same time we've decided at CDM that the preferred name, at least by Core is ExternalReference.

Please check out [this file](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/entity-viewer/types/crossReference.ts) for suggestions.

Reply:
I've merged the `ExternalReference` type with the `CrossReference` type and deleted the `crossReference.ts` file.